ProjectSettings/EditorBuildSettings.asset

```diff
@@ -20,9 +17,6 @@ EditorBuildSettings:
   - enabled: 1
     path: Assets/_BForBoss/_Weapons/Scenes/GunRange.unity
     guid: bc624f0b2db8b7d4286c919435fb2f98
-  - enabled: 1
-    path: Assets/_BForBoss/_LevelDesign/Scenes/CKJ_Weapon_Sandbox.unity
```

Comment:
Was this scene meant to be removed from the editor settings?

Reply:
Woops, adding that back in.